shared/actions/kbfs.desktop.js

```diff
@@ -270,7 +270,7 @@ function* openWithCurrentMountDir(openPath: string): Saga.SagaGenerator<any, any
 
   // turns '/keybase/private/alice' to 'private/alice'
   const subPath = goodPath
-    .split('/')
+    .split(path.sep)
```

Comment:
Hmm isn't this function for KBFS paths only, which always uses forward slashes?

Reply:
path.normalize changes things to backslashes on Windows - maybe that is an error too?
https://millermedeiros.github.io/mdoc/examples/node_api/doc/path.html#path.normalize

Reply:
Ah I see! thanks for the background. Perhaps it'd be better to call `path.posix.normalize` instead of `path.normalize` above so we keep the KBFS path using forward-slashes?

Reply:
that works too